src/pages/RecordingsPage.js

```diff
@@ -0,0 +1,103 @@
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
+import backButton from '../images/back.svg';
+import { nanoid } from 'nanoid';
+
+export default function RecordingsPage({ myRecordings }) {
+  let playerCounter = 0;
+
+  return (
+    <>
+      <HeadingContainer>
+        <BackButton to="/">
+          <img src={backButton} alt="back-button" width="35px" height="35px" />
+        </BackButton>
+        <Heading>My Recordings</Heading>
+      </HeadingContainer>
+      <RecordingsContainer>
+        {myRecordings.map(recording => (
+          <div key={nanoid()}>
```

Comment:
Does the nanoid function do not give the recording a new id every time you map over the array?? I am not sure if this is what you want, but maybe I am wrong or this doesn´t matter for your function :)

Reply:
Thanks you are right, maybe it is better on a other way, but in this case it doesn't matter 😅